inhouse/command_handlers/test_player.py

```diff
@@ -0,0 +1,3 @@
+class TestPlayer:
+  name = "Test"
```

Comment:
Same comment as last PR on this, I think we can subclass this to Player (or remove if you merge it with that stuff)

Reply:
Will rebase to main